### PR TITLE
Add Skyblockchapter

### PR DESCRIPTION
**Pull request by LisonaZ**

### CODE REVIEW

**Comment by Krutoy242** [comment]
Thank you for PR! I will review it later.

**Comment by Krutoy242** [comment]
I see you didnt removed old quests. But its ok, i can do it myself i believe.

**Comment by LisonaZ** [comment]
Yes, previously I moved the old quests to a new chapter, but it always affected other code and made it difficult for me to distinguish what changes I made when submitting a pull request. So, I reworked it, trying not to modify the old quests as much as possible

**Comment by Krutoy242** [comment]
I want to ask you to make some changes and fixes before i could merge that:

- Please remove **all** `Checkmark` tasks. There should be simple tasks like saplings. Players need to automatically getting quests completed without visiting quest page and clicking. Best way is task meaningful item.
  image

- Please remove all empty quests or add reward/tasks to them. There is no point to have quest without tasks or rewards
image

- Instead of making 4 separate item tasks here, you could make 1 item task with 4 `Item List` elements (Right-click task => Edit => Item List)
image

- Remove Market reward since its breaking progression for Shipping Bin. Instead you can add some emerald-generating items like Pam's Bee Queens
image

- Its very hard to collect Apatite Ore in form of block (require silk touch). You need to require Apatite gems instead. Same with other ores like Redstone or Nether Quartz.
image

- Fix names of some quests or remove them
image

- I cant see Cyclic's Scythe quest, please put it back - `bush scythe` is very useful for Ender Lilly getting and `tree scythe` for Ironwood trees.

**Comment by LisonaZ** [comment]
Regarding the first and second points:

For quests like these with all the Checkmark items, my original intention was to link them to the main storyline quests, but I wasn't sure how to set it up better.

For example, with Tinkers' Construct, when I was setting up the quests, I noticed that there were similar quests in the main storyline.

I'm unsure whether I should remake them or link them to the existing quests in the main storyline.
The quest for saplings is also similar.

Regarding the fifth point:
I usually use the Rubber Tinkers' Construct material to craft tools, allowing for precise collection early and easily.
As for the other issues, I will make the modifications.

Regarding points 1 and 2, should they be deleted, remade, or linked to the main storyline? As for point 5, should it still be modified, or is it sufficient to suggest using Rubber to craft tools?

**Comment by Krutoy242** [comment]
> Regarding points 1 and 2, should they be deleted, remade, or linked to the main storyline?

Add appropriate tasks that not copy task from the main chapters. For example, if main one require Wood Logs, you could require Wood Planks. I want to player finish main quest and then, after one or more action could finish Skyblock one. For TCon part builder it could be for example Wooden Shards. And like this.

> use the Rubber Tinkers' Construct material

Me too, but there barely no practical usage for Ore block. So why players should even bother to obtain it?
So, please, do not require players do useless tasks only for the sake of complete quests.

**Comment by Krutoy242** [comment]
> phosphorus ore to convert into Nether tin ore, and I plant Red Orchid on top of redstone ore

You dont need to pick up ore in inventory for this. You could just pour Seared Stone into place and turn it to ore, without mining block.

**Comment by LisonaZ** [comment]
I have changed these quests, but the rewards for them are still not very reasonable

File: config/ftbquests/normal/chapters/08d906a0/0991db53.snbt [hunk 1]
```diff
@@ -11,6 +11,14 @@
 		type: "item",
 		items: [{
 			item: "minecraft:dirt"
-		}]
+		}],
+		count: 8L,
```
Comment (Krutoy242) [inline config/ftbquests/normal/chapters/08d906a0/0991db53.snbt:15]: This quest required 8 dirt blocks, but starting island have 9 dirt blocks. That makes quest requirments too simple, not requiring any machine or even tool. Change count of required dirt to 10.

File: config/ftbquests/normal/chapters/08d906a0/147b1b5f.snbt [hunk 2]
```diff
@@ -25,7 +25,87 @@
 				}
 			}
 		}],
+		consume_items: true,
```
Comment (Krutoy242) [inline config/ftbquests/normal/chapters/08d906a0/147b1b5f.snbt:28]: I like the idea of "upgrading" crook,
Did you tested this in game? Is it working properly?

File: config/ftbquests/normal/chapters/08d906a0/2c4c4fee.snbt [hunk 3]
```diff
@@ -11,5 +11,15 @@
 		items: [{
 			item: "minecraft:emerald"
 		}]
+	}],
+	rewards: [{
+		uid: "fe7357ff",
+		type: "item",
+		item: {
+			id: "ftbquests:lootcrate",
```
Comment (Krutoy242) [inline config/ftbquests/normal/chapters/08d906a0/2c4c4fee.snbt:19]: Please remove all FTB loot crates from Skyblock quests rewards. I believe that player should never receive Loot Crates from optional quests.

**Review by Krutoy242 — CHANGES_REQUESTED** [review]

File: config/ftbquests/normal/chapters/08d906a0/147b1b5f.snbt (hunk 2 above)
Comment (LisonaZ) [reply config/ftbquests/normal/chapters/08d906a0/147b1b5f.snbt:28]: Yes, I have tested it, and it works.

**Review by Krutoy242 — APPROVED** [review]

**Pull request merged**